Dockerfile

```diff
@@ -1,2 +1,2 @@
-FROM docker.pkg.github.com/navikt/pus-nais-java-app/pus-nais-java-app:java11
+FROM docker.pkg.github.com/navikt/pus-nais-java-app/pus-nais-java-app:java17
```

Comment:
I think we're not supposed to use the pus images anymore?
https://github.com/navikt/veilarbportefolje/commit/e2fcf18eae2f901b1ad9571fd756ecb5f3b17889

Reply:
Yeah, they're archived!

Reply:
Actually, it seems we should try to move on from the poao-baseimages as well ([to distroless?](https://sikkerhet.nav.no/docs/sikker-utvikling/containere#dockerfile-eksempler)), because they're building on navikt/baseimages which are not actively maintained!